.github/workflows/dotnetcore.yml

```diff
@@ -30,18 +29,20 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v4.1.1
 
-    - name: Setup .NET 6.0.x
+    - name: Setup .NET 6.x
```

Comment:
6 and 8 shouldn't be needed I don't think? The 9 sdk can built for 6 and 8.

Reply:
Removed.